runtime/compiler/env/VMJ9.cpp

```diff
@@ -8112,7 +8294,58 @@ TR_J9VM::getPrimitiveArrayAllocationClass(J9Class *clazz)
    return (TR_OpaqueClassBlock *) clazz;
    }
 
+#if defined(JITSERVER_SUPPORT)
+TR_StaticFinalData
+TR_J9VM::dereferenceStaticFinalAddress(void *staticAddress, TR::DataType addressType)
```

Comment:
This will need to be used in `J9TransformUtil.cpp` so it needs to stay outside #ifdef (JITSERVER_SUPPORT) like many of the other new frontend queries.